### PR TITLE
Adds KarthikSundar's Blog Post

### PR DESCRIPTION
**Pull request by KarthikSundar2002**

### CODE REVIEW

File: _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+project: CERNBOX
+title: OCM WebDAV Service
+author: Karthik Sundar
+photo: blog_authors/KarthikSundar.jpg
+date: 22.07.2022
+year: 2022
+layout: blog_post
+logo: cernbox-logo.png
+intro: |
+  File Sharing through a OCM WebDAV Service, a Progress Report.
```
Comment (agheata) [inline _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md:11]: Please use an actual introduction in a sentence, this is what people see first before reading your blog.

**Review by agheata — CHANGES_REQUESTED** [review]
Please ask your mentor(s) to review this.

**Comment by labkode** [comment]
@agheata I approve the blogpost, @KarthikSundar2002 you can take @agheata suggestion to have an introduction sentence.

File: _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md (hunk 1 above)
Comment (KarthikSundar2002) [reply _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md]: Yeah, i have changed the intro.

File: _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+project: CERNBOX
+title: OCM WebDAV Service
+author: Karthik Sundar
+photo: blog_authors/KarthikSundar.jpg
+date: 22.07.2022
+year: 2022
+layout: blog_post
+logo: cernbox-logo.png
+intro: |
+  A OCM (Open Cloud Mesh) and WebDAV file sharing service to faciliate sharing of data across the Sciencemesh.
```
Comment (agheata) [inline _gsocblogs/2022/blog_CERNBOX_KarthikSundar.md:11]:
```suggestion
  A OCM (Open Cloud Mesh) and WebDAV file sharing service to facilitate sharing of data across the Sciencemesh.
```

**Pull request merged**